.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,35 +1,41 @@
 ---
-name: Bug report
+name: Bug Report
 about: Create a report to help us improve
-title: ''
-labels: ''
-assignees: ''
-
+title: ""
+labels: "Status: Needs Review, Type: Bug"
+assignees: ""
 ---
 
-### Description
-<!-- Brief description of the bug -->
+<!-- Thank you for taking the time to submit a bug report to The Odin Project. In order to get issues closed in a reasonable amount of time, you must include a baseline of information about the bug in question. Please read this template in its entirety before filling it out to ensure that it is filled out correctly. -->
+
+<!-- Complete the following REQUIRED checkboxes by replacing the whitespace between the square brackets with an 'x', e.g. [x]. -->
+
+-   [ ] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
+-   [ ] The title of this issue follows the `where the change occurs: brief description of bug` format, e.g. `React section: Knowledge Checks don't link to resource`
+
+<!-- The following checkbox is OPTIONAL. Completing it does not guarantee you will be assigned this issue, but rather lets us know you are interested in working on it. -->
+
+-   [ ] I would like to be assigned this issue to work on it
+
+**1. Description of the Bug**
+
+<!-- A clear and concise description of what the bug is. Include any screenshots that may help show the bug in action. -->
+
+**2. How To Reproduce**
+
+<!-- What steps one might need to take in order to reproduce this bug. -->
 
-#### Steps to Reproduce
-<!-- What steps need to be taken in order to reproduce this bug?
+**3. Expected Behavior**
 
-Example:
```

Comment:
Can we keep the example please? it helps prime the bug report author with what format they should provide.

Reply:
Did you want the example *just* for the "how to reproduce" section, or also for the "expected behavior" section?